src/main/java/uk/nhs/hee/trainee/details/interceptor/TraineeIdentityInterceptor.java

```diff
@@ -56,6 +57,14 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
       }
     }
 
+    // Non-CCT endpoints are a mix of authenticated (public) and unauthenticated (internal), limit
+    // trainee ID verification to CCT endpoints for now.
+    if (traineeIdentity.getTraineeId() == null
+        && request.getRequestURI().matches("^/api/cct(/.+)?$")) {
```

Comment:
Yuck. We should clean this up sometime 😀

Reply:
It's definitely a good nudge to move to SQS between sync and details instead of the PATCH endpoints.